doc/INSTALL.md

```diff
@@ -99,3 +99,32 @@ $ nix-shell -Q -p gdb sqlite autoconf git clang libtool gmp sqlite autoconf \
 autogen automake 'python3.withPackages (p: [p.bitcoinlib])' \
 valgrind asciidoc --run make
 ```
+
+To Cross-compile for Android
+--------------------
+
+Make a standalone toolchain as per https://developer.android.com/ndk/guides/standalone_toolchain.html. For c-lightning you must target an API level of 24 or higher.
+
+Depending on your toolchain location and target arch source env variables such as:
```

Comment:
Is this sentence complete? Feels like it is missing a word. `s/source/set/`?

Reply:
It was definitely missing a comma. I've used to word source since I feel people will be using the `source` command on a file rather than setting all these manually.